hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/MultiVersionConcurrencyControl.java

```diff
@@ -185,7 +191,7 @@ public void completeAndWait(WriteEntry e) {
    */
   public boolean complete(WriteEntry writeEntry) {
     synchronized (writeQueue) {
-      writeEntry.markCompleted();
+      writeEntry.markCompleted(true);
```

Comment:
Why pass the boolean?

Reply:
maybe should change the function name from `markCompleted` to `setCompleted`

Reply:
Do we ever pass false? I don't see it (perhaps we do). If we don't, then markCompleted seems fine (without a param). setCompleted makes sense if a param is passed.

Reply:
Yes,will use it in the place:
`WriteEntry e = cachedWriteEntries.get().setWriteNumber(nextWriteNumber).markCompleted(false);`
Let me use `setCompleted` instead